content/ember/v5/deprecate-array-prototype-extensions.md

```diff
@@ -6,6 +6,16 @@ since: '5.10.0'
 
 Ember historically extended the prototypes of native Javascript arrays to implement `Ember.Enumerable`, `Ember.MutableEnumerable`, `Ember.MutableArray`, `Ember.Array`. As of v5, the usages of array prototype extensions are deprecated.
 
+To disable the extention of array prototypes, in `config/environment.js`, ensure that `EXTEND_PROTOYPES` is set to false on EmberENV:
+
+```
+EmberENV: {
+  EXTEND_PROTOTYPES: false,
+....
+```
```

Comment:
Use code fence language for md lint:

```suggestion
```js
EmberENV: {
  EXTEND_PROTOTYPES: false,
  // ...
},
// ...
```
```